src/components/MainPage/MainPage.module.css

```diff
@@ -0,0 +1,122 @@
+@media (min-width: 1084px) {
+    .intro {
+        position: relative;
+    }
+
+    .introContent {
+        margin-right: 632px;
+    }
+
+    .intro::after {
+        display: block;
+        content: '';
+        position: absolute;
+        top: 0;
+        right: 0;
+        width: 632px;
+        height: 798px;
+        background: url("./logo.png") no-repeat top center;
+        background-size: 130%;
+    }
+}
+
+.companies {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    list-style: none;
+    padding: 0;
+    gap: 16px 94px;
```

Comment:
A few months ago we decided not to use `flex-gap` due to not great browser support, are we want to use it now?